### PR TITLE
SCALRCORE-16917: Use Revizor API to create test containers to run acceptance tests against

### PR DESCRIPTION
**Pull request by vmotso**
Use Revizor API to create test containers to run acceptance tests against
SCALRCORE-16917

### Changelog
* [ ] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -15,14 +15,53 @@ jobs:
...
+        with:
+          command: create
+      - name: Run acceptance tests
+        env:
+          SCALR_HOSTNAME: ${{ steps.create.outputs.hostname }}
+        run: make testacc
+      - name: Delete container
+        id: delete
+        if: ${{ steps.create.outputs.container_id }}
+        uses: Scalr/gh-action-revizor@feature/SCALRCORE-16917
```
Comment (vmotso) [inline .github/workflows/default.yml:64]: After CR - I will change to
```
Scalr/gh-action-revizor@feature/SCALRCORE-16917 ->
Scalr/gh-action-revizor@master
```

File: scalr/data_source_endpoint_test.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func testAccEndpointDataSourceConfig(rInt int) string {
 	return fmt.Sprintf(`
 resource scalr_environment test {
   name       = "test-env-%[1]d"
-  account_id = "existing"
+  account_id = "acc-svrcncgh453bi8g"
```
Comment (lbn) [inline scalr/data_source_endpoint_test.go:44]: Can we put this account ID into a const?

**Review by lbn — CHANGES_REQUESTED** [review]

File: .github/workflows/default.yml [hunk 3]
```diff
@@ -15,14 +15,53 @@ jobs:
...
+        with:
+          command: create
+      - name: Run acceptance tests
+        env:
+          SCALR_HOSTNAME: ${{ steps.create.outputs.hostname }}
+        run: make testacc
+      - name: Delete container
+        id: delete
+        if: ${{ steps.create.outputs.container_id }}
```
Comment (lbn) [inline .github/workflows/default.yml:63]: Have you confirmed that this step runs even if the tests fail?

Comment (vmotso) [reply .github/workflows/default.yml]: Now it works.
See checks 36, 37
https://github.com/vmotso/check-gh-action-revizor/actions

File: scalr/testing.go [hunk 4]
```diff
@@ -7,6 +7,8 @@ import (
 	scalr "github.com/scalr/go-scalr"
 )
 
+const DefaultAccount = "acc-svrcncgh453bi8g"
```
Comment (lbn) [inline scalr/testing.go:10]: Need to make it private just like everything else we use in tests.

**Review by lbn — CHANGES_REQUESTED** [review]

**Review by lbn — APPROVED** [review]

**Pull request merged**